vivisect/__init__.py

```diff
@@ -54,7 +54,7 @@
 
 
 def guid(size=16):
-    return binascii.hexlify(os.urandom(size))
+    return binascii.hexlify(os.urandom(size)).decode()
```

Comment:
decode as what? recommend explicit encoding.

sidebar: how does this even work???

```suggestion
    return binascii.hexlify(os.urandom(size)).decode("ascii")
```

Reply:
I usually play pretty fast and loose with python defaults, so in this case, python3 (according to their docs), defaults to utf-8. So we don't *typically* need to actually define it.

Though as @atlas0fd00m pointed out, it's pretty easy to override with env vars. So explicit is better in this case. Old habits just die hard :smile: